vmm/src/config.rs

```diff
@@ -2301,6 +2329,59 @@ impl TpmConfig {
     }
 }
 
+impl LandlockConfig {
+    pub const SYNTAX: &'static str = "Landlock parameters \
+        \"path=<path/to/{file/dir}>,flags={r,w,x}\". Both path and flags options are required.\"";
+
+    pub fn parse(landlock_rule: &str) -> Result<Self> {
+        let mut parser = OptionParser::new();
+        parser.add("path").add("flags");
+        parser
+            .parse(landlock_rule)
+            .map_err(Error::ParseLandlockRules)?;
+
+        let path = parser
+            .get("path")
+            .map(PathBuf::from)
+            .ok_or(Error::ParseLandlockMissingFields)?;
+
+        let mut path_perms = landlock::Perms::empty();
+        let flags = parser
+            .get("flags")
```

Comment:
I suggest using "access" instead of "flags" (here and in the doc), which is self-explanatory.